src/ctlib/ct.c

```diff
@@ -496,10 +496,19 @@ ct_con_props(CS_CONNECTION * con, CS_INT action, CS_INT property, CS_VOID * buff
 		case CS_TIMEOUT:
 			/* set the query timeout as an integer in seconds */
 		        tds_login->query_timeout = *(CS_INT *) buffer;
+			if (tds_login->query_timeout == CS_NO_LIMIT) {
```

Comment:
style: no bracket for a single line

Reply:
Ah, right, we use different style conventions, and I evidently forgot to flip back to your style here.